packages/commonwealth/server/controllers/server_proposal_methods/get_completed_proposals.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  ServerProposalsController,
+  SupportedProposalNetworks,
+} from '../server_proposals_controller';
+import { providers } from 'ethers';
+import { ChainNetwork } from 'common-common/src/types';
+import { ServerError } from 'common-common/src/errors';
+import { formatAaveProposal, getEthereumAaveProposals } from './aave/proposals';
+
+export type GetCompletedProposalsOptions = {
+  chainId: string;
+};
+
+export type GetCompletedProposalsResult = {
+  completedProposals: any[];
+};
+
+export async function __getCompletedProposals(
+  this: ServerProposalsController,
+  { chainId }: GetCompletedProposalsOptions,
+  provider: providers.Web3Provider,
+  contractInfo: { address: string; type: SupportedProposalNetworks }
+) {
```

Comment:
Should have explicit return type in function signature (`Promise<GetCompletedProposalsResult>`)

Reply:
Added.